src/svm/programConnectors.ts

```diff
@@ -0,0 +1,35 @@
+import { Idl, Program, Provider } from "@coral-xyz/anchor";
+import {
+  MessageTransmitterAnchor,
+  MessageTransmitterIdl,
+  MulticallHandlerAnchor,
+  MulticallHandlerIdl,
+  SvmSpokeAnchor,
+  SvmSpokeIdl,
+  TokenMessengerMinterAnchor,
+  TokenMessengerMinterIdl,
+} from "./assets";
+
+export function getConnectedProgram<P extends Idl>(idl: P, provider: Provider, programId?: string) {
+  if (programId) {
+    // An IDL must have been previously initialized
+    return Program.at<P>(programId, provider);
```

Comment:
Note: In the SVM scripts, we have been connecting to the program ID defined in the SvmSpokeIdl file. Since these IDs are generated automatically, any future changes to the program, if merged into master, could cause the program ID to drift.
I've added the optional programId argument to target specific id's and I think this is safer I general. Wdy?